examples/3box/index.js

```diff
@@ -0,0 +1,10 @@
+const IdentityWallet = require('identity-wallet');
+
+let threeIdProvider;
+wallet.registerRpcMessageHandler(async (origin, req) => {
+  if (!threeIdProvider) {
+    const idWallet = new IdentityWallet({ seed: '0x' + (await wallet.getAppKey()) });
+    threeIdProvider = idWallet.get3idProvider();
+  }
+  return threeIdProvider.send(req, origin);
+});
```

Comment:
Love that this is literally 10 lines. We'll provide an `.eslintrc` in the future, but can you please remove all semicolons for stylistic consistency?